tests/loading_data_test.py

```diff
@@ -240,53 +191,14 @@ def test_load_adult_race():
     assert "salary" not in adult_race.disc_feature_groups
 
 
-def test_load_adult_race_binary():
-    """test load adult race binary"""
-    data: DataTuple = adult("Race-Binary").load()
-    assert (45222, 98) == data.x.shape
-    assert (45222, 1) == data.s.shape
-    assert (45222, 1) == data.y.shape
-    assert data.name == "Adult Race-Binary"
-
-
-def test_load_compas_race():
-    """test load compas race"""
-    data: DataTuple = compas("Race").load()
-    assert (6167, 400) == data.x.shape
-    assert (6167, 1) == data.s.shape
-    assert (6167, 1) == data.y.shape
-
-
 def test_load_adult_race_sex():
-    """test load adult race sex"""
+    """Test load of split combinatin that doesn't exist yet."""
```

Comment:
ah, this isn't a test of a split combination that doesn't exist. the problem here is something else: it's a multi s setup that's very complicated, because it's combining a binary attribute (sex) with a one-hot encoded attribute (race) which requires some complicated code to make it work. I hope that one day, this doesn't throw an assertion error.

Reply:
aha, I'll revert the comment. Multi S is tricky. How do big projects discuss this sort of thing? With a proposal?

Reply:
yes! write a proposal!